$OpenBSD: patch-Modules_GNUInstallDirs_cmake,v 1.6 2014/09/30 07:14:49 dcoppa Exp $

commit d4fdd9c189f85d659f4294f8ec6da3e7e51215ec
Author: Rolf Eike Beer <eike@sf-mail.de>
Date:   Tue May 20 18:14:10 2014 +0200

GNUInstallDirs: use the proper default for info and man paths on OpenBSD

commit 33f031295c2bb8d652e4d5f8749e50ef24d7adb6
Author: David Coppa <dcoppa@gmail.com>
Date:   Mon Sep 29 12:19:12 2014 +0200

Fix typo in Modules/GNUInstallDirs.cmake (#15176)

Commit d4fdd9c189f85d659f4294f8ec6da3e7e51215ec ("GNUInstallDirs: use the
proper default for info and man paths on OpenBSD") introduced a typo.

--- Modules/GNUInstallDirs.cmake.orig	Thu Sep 11 15:24:00 2014
+++ Modules/GNUInstallDirs.cmake	Mon Sep 29 10:37:17 2014
@@ -192,19 +192,31 @@ if(NOT CMAKE_INSTALL_DATADIR)
   set(CMAKE_INSTALL_DATADIR "${CMAKE_INSTALL_DATAROOTDIR}")
 endif()
 
-if(NOT CMAKE_INSTALL_INFODIR)
-  set(CMAKE_INSTALL_INFODIR "" CACHE PATH "info documentation (DATAROOTDIR/info)")
-  set(CMAKE_INSTALL_INFODIR "${CMAKE_INSTALL_DATAROOTDIR}/info")
+if(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  if(NOT CMAKE_INSTALL_INFODIR)
+    set(CMAKE_INSTALL_INFODIR "" CACHE PATH "info documentation (info)")
+    set(CMAKE_INSTALL_INFODIR "info")
+  endif()
+
+  if(NOT CMAKE_INSTALL_MANDIR)
+    set(CMAKE_INSTALL_MANDIR "" CACHE PATH "man documentation (man)")
+    set(CMAKE_INSTALL_MANDIR "man")
+  endif()
+else()
+  if(NOT CMAKE_INSTALL_INFODIR)
+    set(CMAKE_INSTALL_INFODIR "" CACHE PATH "info documentation (DATAROOTDIR/info)")
+    set(CMAKE_INSTALL_INFODIR "${CMAKE_INSTALL_DATAROOTDIR}/info")
+  endif()
+
+  if(NOT CMAKE_INSTALL_MANDIR)
+    set(CMAKE_INSTALL_MANDIR "" CACHE PATH "man documentation (DATAROOTDIR/man)")
+    set(CMAKE_INSTALL_MANDIR "${CMAKE_INSTALL_DATAROOTDIR}/man")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_LOCALEDIR)
   set(CMAKE_INSTALL_LOCALEDIR "" CACHE PATH "locale-dependent data (DATAROOTDIR/locale)")
   set(CMAKE_INSTALL_LOCALEDIR "${CMAKE_INSTALL_DATAROOTDIR}/locale")
-endif()
-
-if(NOT CMAKE_INSTALL_MANDIR)
-  set(CMAKE_INSTALL_MANDIR "" CACHE PATH "man documentation (DATAROOTDIR/man)")
-  set(CMAKE_INSTALL_MANDIR "${CMAKE_INSTALL_DATAROOTDIR}/man")
 endif()
 
 if(NOT CMAKE_INSTALL_DOCDIR)
